src/js/components/AlertDialogComponent.jsx

```diff
@@ -0,0 +1,72 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+
+var Util = require("../helpers/Util");
+var ModalComponent = require("../components/ModalComponent");
+
+var AlertModalComponent = React.createClass({
+  displayName: "AlertModalComponent",
+
+  propTypes: {
+    data: React.PropTypes.shape({
+      actionButtonLabel: React.PropTypes.string.isRequired,
+      message: React.PropTypes.string.isRequired,
+      state: React.PropTypes.string.isRequired,
```

Comment:
I was also thinking about using a different term due to the same reasons and only used it, as it used within the related issue description. But yeah, let's rename it to "level".